src/base/users/View.svelte

```diff
@@ -0,0 +1,92 @@
+<script lang="typescript">
+  import { onMount } from 'svelte';
+  import { navigate } from 'svelte-routing';
+  import type { Config } from '@app/config';
+  import type { Registration } from '@app/base/registrations/registrar';
+  import { getRegistration } from '@app/base/registrations/registrar';
+  import Icon from '@app/Icon.svelte';
+  import Loading from '@app/Loading.svelte';
+  import Address from '@app/Address.svelte';
+  import Avatar from '@app/Avatar.svelte';
+  import Error from '@app/Error.svelte';
+
+  export let address: string;
+  export let config: Config;
+  
+  let addressName: string | null = null;
+  let getInfo: Promise<Registration | null>;
+  
+  onMount(async () => {
+    addressName = await config.provider.lookupAddress(address);
+    getInfo = getRegistration(addressName, config);
+  });
+</script>
+
+<style>
+  main {
+    padding: 5rem 0;
+  }
+  main > header {
+    display: flex;
+    align-items: center;
+    justify-content: left;
+    margin-bottom: 2rem;
+  }
+  main > header > * {
+    margin: 0 1rem 0 0;
+  }
+  .info {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: left;
+  }
+  .info a {
+    border: none;
+  }
+  .avatar {
+    width: 64px;
+    height: 64px;
+  }
+  .links {
+    display: flex;
+    align-items: center;
+    justify-content: left;
+  }
+  .url {
+    display: flex; /* Ensures correct vertical positioning of icons */
+    margin-right: 1rem;
+  }
+</style>
+
+{#await getInfo}
```

Comment:
Here too I would try to move `getInfo` down, so that we can progressively render the page, otherwise if getInfo fails, nothing renders at all.

Reply:
I think you can have multiple `await` blocks on the same promise, though I've never tried.

Reply:
> I think you can have multiple `await` blocks on the same promise, though I've never tried.

I'll give that a try since I had to compromise on what I wanted to be awaited on..

Reply:
I removed the waiting on a result, there is some flickering while the information loads, but it's acceptable.